prod/westeurope/internal/api/functions_services_r3/function_app/terragrunt.hcl

```diff
@@ -66,7 +66,7 @@ inputs = {
   app_service_plan_info = {
     kind     = "elastic"
     sku_tier = "ElasticPremium"
-    sku_size = "EP3"
+    sku_size = "EP1"
   }
 
   runtime_version = "~3"
```

Comment:
Maybe here we can specify the pre_warmed_instance_count like functions_bonusapi_r3
On Azure now the pre_warmed_instance_count is set to 1 for io-p-fn3-services